pkg/endpointmanager/manager.go

```diff
@@ -802,3 +804,22 @@ func (mgr *endpointManager) GetEndpointNetnsCookieByIP(ip netip.Addr) (uint64, e
 
 	return ep.NetNsCookie, nil
 }
+
+// UpdatePolicy triggers policy updates for all live endpoints.
+// Endpoints with security IDs in provided set will be regenerated. Otherwise, the endpoint's
+// policy revision will be bumped to toRev.
+func (mgr *endpointManager) UpdatePolicy(idsToRegen *set.Set[identity.NumericIdentity], fromRev, toRev uint64) {
+	eps := mgr.GetEndpoints()
+
+	wg := sync.WaitGroup{}
+	wg.Add(len(eps))
+
+	for _, ep := range eps {
+		go func(ep *endpoint.Endpoint) {
```

Comment:
The previous version of this was also unbounded. Given that the number of endpoints is also relatively bounded, I think it's not necessary.

Happy to be overruled though.

Reply:
I just remembered a `TODO bound by number of CPUs` somewhere and thought this may be a good moment to do so if we still think it is a good idea. Would probably prevent us from kicking off too many BPF compilations at the same time, leading to resource starvation (and deadline exceeded errors) for clang etc?

Reply:
There already is resource management for regenerations; this is just one goroutine waiting for endpoint regeneration to proceed. I don't think bounding by CPUs makes a difference here.